lib/manageiq/environment.rb

```diff
@@ -1,6 +1,3 @@
-require 'fileutils'
-require 'pathname'
```

Comment:
This file was intended to be a standalone file that was used (potentially) outside of a Rails context.  So unless the sample console calls are just from a vanilla `irb` session, I am 👎 on these changes.

Also, generally speaking, `require "..."` calls are free if they have been called previously, and are essentially a no-op.  We also do "deferred" requires a lot inside methods, and those get called multiple times throughout the app lifecycle and **"usually"** don't cause problem (except that one time where it caused a memory leak...).